### DISCUSSION TITLE
iOS app crash as soon as   let realm = try! Realm() this executes

### DISCUSSION THREAD

**Discussion by rajaMusharaf** [realm/realm-cocoa#8639] [Q&A]
while initilizing realm in iOS 12 app crashes and goes to code 
static void create_empty_group_when_missing(Group& group)
    {
        if (!group.m_top.is_attached())
            group.create_empty_group(); // Throws
    }
<img width="1141" alt="Screenshot 2024-06-26 at 10 51 45 AM" src="https://github.com/realm/realm-swift/assets/98593594/ebd62644-43d2-499e-a4dd-5805ef48a36c">

**Comment by nirinchev** [comment]
Please avoid opening multiple issues for the same problem.
